entropylab/api/param_store.py

```diff
@@ -73,6 +71,21 @@ def list_commits(self, label):
         """
         pass
 
+    @abstractmethod
+    def list_values(self, key: str) -> List[Tuple[Any, datetime, str, str]]:
```

Comment:
@urig I think that the return type here should be of a type that is easily plotted. maybe return a tuple of lists? or a numpy array?
worth adding a test that cheks you can get just the list of the values without the metadata...

@galwiner ?

Reply:
An easy way to get what Guy is suggesting is the following: 
```python
results=pd.DataFrame([(1,2,'c',4),(2,3,'a','b'),(10,3,'a','b'),(20,3,'a','b')])
```
this then allows to slice through: 
```
results[0]
```
an even better experience is obtained if we give text names to the data frame columns

Reply:
`list_values()` now returns a `pd.DataFrame` with named columns. 👍